Sources/MapboxCoreNavigation/EHorizon/ElectronicHorizonDelegate.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import MapboxNavigationNative
+
+public protocol ElectronicHorizonDelegate: class {
+
+    /**
+     Might be called multiple times when the position changes
+
+     - parameter position: Current electronic horizon position (map matched position + e-horizon tree)
+     - parameter distances: Map road object id -> RoadObjectDistanceInfo for upcoming road objects
+     */
+    func didUpdatePosition(_ position: ElectronicHorizonPosition, distances: [String : RoadObjectDistanceInfo])
```

Comment:
Create a type alias for strings and use it for the keys of this dictionary. RoadObjectDistanceInfo will need to be converted to a Swift struct; fortunately that seems to be more straightforward than converting ElectronicHorizonPosition to something more usable by the application.

Reply:
Yes, done.